x-pack/plugin/analytics/src/main/java/org/elasticsearch/xpack/analytics/topmetrics/TopMetricsAggregationBuilder.java

```diff
@@ -169,6 +170,20 @@ protected void doWriteTo(StreamOutput out) throws IOException {
         out.writeList(metricFields);
     }
 
+    @Override
+    public TopMetricsAggregationBuilder subAggregations(Builder subFactories) {
+        throw new AggregationInitializationException(
+            "Aggregator [" + name + "] of type [" + getType() + "] cannot accept sub-aggregations"
+        );
+    }
+
+    @Override
+    public TopMetricsAggregationBuilder subAggregation(AggregationBuilder aggregation) {
+        throw new AggregationInitializationException(
```

Comment:
I think this is not needed and can be removed.

Reply:
Thanks, updated